src/components/ColorChoiceDialog.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { withTheme } from "@material-ui/core/styles";
+
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import Grid from '@material-ui/core/Grid';
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import { ChromePicker } from 'react-color'
+import ResponsiveSetCard from "./ResponsiveSetCard";
+
+function ColorChoiceDialog(props) {
+  const { open, onClose, title, theme } = props;
+
+  const [red, setRed] = useState(theme.setCard.red);
+  const [green, setGreen] = useState(theme.setCard.green);
+  const [purple, setPurple] = useState(theme.setCard.purple);
+
+  function handleClose() {
+    onClose(null);
+  }
+
+  function handleSubmit() {
+    onClose({red: red, green: green, purple: purple});
```

Comment:
Example of code that is formatted differently than the rest of the codebase, where Prettier is used.